modules/op-installing-tkn-on-linux-using-rpm.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/tkn_cli/installing-tkn.adoc
+
+[id="installing-tkn-on-linux-using-rpm"]
+
+= Installing {pipelines-title} CLI (tkn) on Linux using an RPM
+
+For {op-system-base-full} version 8, you can install the {pipelines-title} CLI (`tkn`) as an RPM if you have an active {product-title} subscription on your Red Hat account.
```

Comment:
Should this info be mentioned in the prerequisites? (It's kind of a double-kill to mention it twice but an active subscription is, technically, a prerequisite in this case) WDYT @Preeticp @nkakkar81 ?

Reply:
How about adding it only to the prerequisite section? It will avoid repetition and put it in a place the user can not ignore.

Reply:
Have added in Pre-req section.